fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/flaps_computer.rs

```diff
@@ -695,6 +711,68 @@ mod tests {
         }
     }
 
+    struct TestAdirs {
+        is_aligned: bool,
+        latitude: Arinc429Word<Angle>,
+        longitude: Arinc429Word<Angle>,
+        heading: Arinc429Word<Angle>,
+        vertical_speed: Arinc429Word<Velocity>,
+        altitude: Arinc429Word<Length>,
+        angle_of_attack: Arinc429Word<Angle>,
+    }
+    impl TestAdirs {
+        fn new() -> Self {
+            Self {
+                is_aligned: false,
+                latitude: Arinc429Word::new(Angle::default(), SignStatus::FailureWarning),
+                longitude: Arinc429Word::new(Angle::default(), SignStatus::FailureWarning),
+                heading: Arinc429Word::new(Angle::default(), SignStatus::FailureWarning),
+                vertical_speed: Arinc429Word::new(Velocity::default(), SignStatus::FailureWarning),
+                altitude: Arinc429Word::new(Length::default(), SignStatus::FailureWarning),
+                angle_of_attack: Arinc429Word::new(
+                    Angle::new::<degree>(0.0),
+                    SignStatus::NormalOperation,
+                ),
+            }
+        }
+
+        fn set_angle_of_attack(&mut self, v: Angle) {
+            self.angle_of_attack = Arinc429Word::new(v, SignStatus::NormalOperation);
+        }
+    }
```

Comment:
This feels very weird. Don't we already have the stuff to mock the ADR in the test bed? @Gurgel100 will know better than me.

Reply:
I looked everywhere, could only find similar boilerplate code everywhere, but no re-usable test bed. Don't know how it's done in the Rust world

Reply:
Hmm, we have a comprehensive ADIRU test bed but it seems it's not public. I withdraw the comment here as out of scope, but I think there's an opportunity to avoid having many copies of the test bed setup code in future.